android/examples/snippets/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1 @@
+<manifest package="com.mongodb.stitch.android.services.mongodb.remote" />
```

Comment:
[q] should this be be `"com.mongodb.stitch.android.examples.snippets"`?

Reply:
Yep, good catch!